dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/CompanyManager.kt

```diff
@@ -115,6 +117,12 @@ class CompanyManager(
         filter: CompanySearchFilter,
         viewingUser: DatalandAuthentication? = null,
     ): List<StoredCompany> {
+        if (filter.dataTypeFilter.isEmpty()) {
+            throw InvalidInputApiException(
+                "Requestparam has a non acceptable value",
+                "Please specify a dataframework",
+            )
```

Comment:
Das müsste dann wieder rausgenommen werden, wenn wir leere dataType Filter erlauben, oder?

Reply:
I modified to set all datatypes as filter if none are set